torch/__init__.py

```diff
@@ -170,6 +170,7 @@ def _load_global_deps():
             import torch._dl as _dl_flags  # type: ignore
     old_flags = sys.getdlopenflags()
     sys.setdlopenflags(_dl_flags.RTLD_GLOBAL | _dl_flags.RTLD_LAZY)
+    # Note that this import ignores all the functions whose name that starts with "_"
```

Comment:
nit: link to where this is documented in the python docs?

Reply:
It is not very well documented. But mentioned here: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package
Will add a link to this in the comment.